test/test_other.py

```diff
@@ -2875,6 +2875,17 @@ def test_jspi_wildcard(self):
 
     self.do_runf(test_file('other/test_jspi_wildcard.c'), 'done')
 
+  def test_jspi_addfunction(self):
+    # make sure synchronous functions in the wasmTable aren't processed with Asyncify.makeAsyncFunction
+    self.require_v8()
```

Comment:
This can be `self.require_jspi()` and then you can get rid of v8 args below.

Reply:
done